spec/truffle/interop/special_forms_spec.rb

```diff
@@ -70,6 +70,34 @@
     pa.log.should include([:polyglot_read_array_element, 0])
   end
 
+  it description['.at(index)', :at, (:index)] do
+    pfo, pa, l = proxy[TruffleInteropSpecs::PolyglotArray.new]
+    -> { pfo.at 0 }.should raise_error(IndexError)
+    l.log.should include(['getArraySize'])
+    pa.log.should include([:polyglot_array_size])
```

Comment:
These should check for `readArrayElement` since that's the primary interop message used.
Same below of course.